### PR TITLE
update benchmark/v5.1-performance-benchmarking-with-tpcc.md,benchmark…

### PR DESCRIPTION
**Pull request by Johnny2020-Chen**
…/v5.2-performance-benchmarking-with-tpcc.md,benchmark/v5.3-performance-benchmarking-with-tpcc.md,two-data-centers-in-one-city-deployment.md

### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [X] master (the latest development version)
- [X] v5.3 (TiDB 5.3 versions)
- [X] v5.2 (TiDB 5.2 versions)
- [X] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s): https://github.com/pingcap/docs-cn/issues/7735

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by Johnny2020-Chen** [comment]
signed Contributor License Agreement

File: benchmark/v5.2-performance-benchmarking-with-tpcc.md [hunk 1]
```diff
@@ -81,7 +81,7 @@ set global tidb_enable_clustered_index = 1;
 
 1. 通过 TiUP 部署 TiDB v5.2.0 和 v5.1.1。
 2. 创建数据库 tpcc：`create database tpcc;`
-3. 通过 tiup bench 导入 TPC-C 5000 Warehouse 数据：`tiup bench tpcc prepare --warehouse 5000 --db tpcc -H 127.0.0.1 -p 4000`。
+3. 通过 tiup bench 导入 TPC-C 5000 Warehouse 数据：`tiup bench tpcc prepare --warehouses 5000 --db tpcc -H 127.0.0.1 -p 4000`。
```
Comment (TomShawn) [inline benchmark/v5.2-performance-benchmarking-with-tpcc.md:84]: @Yui-Song PTAL. Thanks!

**Review by Yui-Song — APPROVED** [review]
LGTM

**Review by lucklove — APPROVED** [review]
LGTM

File: two-data-centers-in-one-city-deployment.md [hunk 2]
```diff
@@ -82,7 +82,7 @@ alertmanager_servers:
 
 ### Placement Rules 规划
 
-为了按照规划的集群拓扑进行部署，你需要使用 [Placement Rules](/configure-placement-rules.md) 来规划集群副本的放置位置。以 4 副本和 2 Voter 副本在主数据中心，1 Voter 和 1 Learner 在从数据中心的部署方式为例，可使用 Placement Rules 进行如下副本配置：
+为了按照规划的集群拓扑进行部署，你需要使用 [Placement Rules](/configure-placement-rules.md) 来规划集群副本的放置位置。以 4 副本(2 Voter 副本在主数据中心，1 Voter 和 1 Learner 在从数据中心)的部署方式为例，可使用 Placement Rules 进行如下副本配置：
```
Comment (TomShawn) [inline two-data-centers-in-one-city-deployment.md:85]:
```suggestion
为了按照规划的集群拓扑进行部署，你需要使用 [Placement Rules](/configure-placement-rules.md) 来规划集群副本的放置位置。以 4 副本（2 Voter 副本在主数据中心，1 Voter 和 1 Learner 在从数据中心）的部署方式为例，可使用 Placement Rules 进行如下副本配置：
```

**Review by TomShawn — APPROVED** [review]
LGTM

**Pull request merged**